jme3-effects/src/main/java/com/jme3/water/WaterFilter.java

```diff
@@ -292,8 +303,20 @@ protected void cleanUpFilter(Renderer r) {
     @Override
     public void write(JmeExporter ex) throws IOException {
         super.write(ex);
+
         OutputCapsule oc = ex.getCapsule(this);
 
+        final Spatial reflectionScene = getReflectionScene();
+        final boolean needSaveReflectionScene = isNeedSaveReflectionScene();
+
+        oc.write(causticsTexture, "causticsTexture", null);
+        oc.write(heightTexture, "heightTexture", null);
+        oc.write(normalTexture, "normalTexture", null);
+        oc.write(foamTexture, "foamTexture", null);
+        oc.write(needSaveReflectionScene, "needSaveReflectionScene", false);
+        oc.write(needSaveReflectionScene ? reflectionScene : null, "reflectionScene",
```

Comment:
This may break j3o/j3f backward compatibility.
Some people may have already saved post processes without the reflection scene. It would be nice to make sure those files still loads after this change. (to test it use the sdk and save a j3f file with a water filter, then try to load it in the engine)